extensions/ql-vscode/src/remote-queries/run-remote-query.ts

```diff
@@ -583,9 +583,13 @@ function buildRemoteQueryEntity(
     return { owner: owner, name: repo };
   });
 
+  // TODO: Get query text from query file and save it in a temporary .ql file. 
+  const queryTextTmpFile = '';
```

Comment:
Might be worth writing a file here with some dummy text to help with our testing (e.g. we'll see the file opening, it just won't have the full text).